src/main/java/competition/subsystems/drive/commands/vision_path/DriveVectorSmallCommand.java

```diff
@@ -0,0 +1,97 @@
+package competition.subsystems.drive.commands.vision_path;
+
+import competition.subsystems.drive.DriveSubsystem;
+import competition.subsystems.pose.PoseSubsystem;
+import edu.wpi.first.math.geometry.Pose2d;
+import xbot.common.command.BaseCommand;
+import xbot.common.controls.sensors.XTimer;
+import xbot.common.math.XYPair;
+
+import javax.inject.Inject;
+
+public class DriveVectorSmallCommand extends BaseCommand {
+    private double start;
+    private double last = 0.25;
```

Comment:
⭐ consider calling this `duration` and making it of type `Time` so the units can't be messed up
```suggestion
    private Time duration = Seconds.of(0.25);